### ISSUE TITLE
Bug: Can't copy part or some lines during reviewing unstaged files

### ISSUE THREAD

**Issue by tamvm** [rowanj/gitx#113]
I have checked on many versions, this bug still exists. Don't know it's your intention or not but sometimes, when review code before commit, we need to copy part of code or few lines but there no way. 

Hope this issue is fixed soon.

**Comment by rowanj** [comment]
Typically this happens if the output of the `diff` utility includes lines that can't be parsed; for example if there is no newline at the end of the file, the diff output includes that, which can't (at least, shouldn't) be inserted into your changeset.

I've played around with stripping such lines within GitX, but I'm hesitant because of the fact that it can't be distinguished from actual input.

**Comment by tamvm** [comment]
Yes I see, but this issue doesn't happen when reviewing old commits, so hope that you will find the good solution for this issue as well.

**Comment by rowanj** [comment]
I think I added that comment to the wrong issue; yes - we should add the ability to copy from the "Stage" view.

**Comment by tamvm** [comment]
Yes, waiting for next release :-)

**Comment by joehoyle** [comment]
+1
